angular-portfolio/src/app/award-detail/award-detail.component.ts

```diff
@@ -0,0 +1,39 @@
+import { Component, OnInit, Input } from '@angular/core';
+import { Awards } from '../awardsTemplate';
+import { AwardsService } from '../awards.service';
+
+@Component({
+  selector: 'app-award-detail',
+  templateUrl: './award-detail.component.html',
+  styleUrls: ['./award-detail.component.scss']
+})
+export class AwardDetailComponent implements OnInit {
+
+  awards: Awards[];
+  @Input() award: Awards;
+  hoverIndex: number;
+  current:  Awards;
+
+  constructor(private awardService: AwardsService) { }
+
+  ngOnInit(): void {
+    this.getAwards();
+  }
+
+  getAwards(): void {
+    this.awardService.getAwards()
+      .subscribe(awards => this.awards = awards);
+  }
+
+  enter(current: Awards) {
```

Comment:
Since you've included the return type of "void" for the preceding two methods, please also include it for enter and exit. Whether or not to include return types for methods is a matter of individual style, but it should be consistent across the app.